skrub/_dataframe/_common.py

```diff
@@ -824,6 +826,25 @@ def _to_categorical_polars(col):
     col = to_string(col)
     return _cast_polars(col, pl.Categorical())
 
+@dispatch
+def is_all_null(col):
+    raise NotImplementedError()
+
+
+@is_all_null.specialize("pandas", argument_type="Column")
+def _is_all_null_pandas(col):
+    return bool(col.isna().all())
```

Comment:
```suggestion
    return col.isna().all()
```

I think it's already a boolean here

Reply:
`col.isna().all()` of type `numpy.bool`, I'm not sure if that makes a difference downstream, I decided to cast it to bool to avoid issues.

Reply:
I think that's a good idea